api/events/content.proto

```diff
@@ -23,6 +23,10 @@ import "github.com/containerd/containerd/api/types/fieldpath.proto";
 option go_package = "github.com/containerd/containerd/api/events;events";
 option (containerd.types.fieldpath_all) = true;
 
+message ContentCreate {
+	string digest = 1;
```

Comment:
We should include size here as well

It would be nice to include size on `ContentDelete`, but that would cause extra effort on Delete that isn't worth it. The Commit calls will have the size anywhere it has the digest.

Reply:
We could probably add it to the delete event but it would require calling info on every blob we delete. It might be good to leave that for a separate PR?